tests/main/interfaces-shutdown-introspection/task.yaml

```diff
@@ -1,11 +1,8 @@
 summary: Ensures that introspection of login1 of the shutdown interface works.
 
-systems:
-    # No confinement (AppArmor, Seccomp) available on these systems
-    - -debian-*
-    # unity7 implicit classic slot needed (used to access dbus-send) not
-    # available on core
-    - -ubuntu-core-*
+# No confinement (AppArmor, Seccomp) available on these systems
```

Comment:
I think we should expand the comment a little bit, e.g. "debian: no confinement..." and "ubuntu-core-*: unity7 implict ..."

Reply:
sure, thanks for reviewing this